catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/fragment/ScriptFragmentTest.java

```diff
@@ -237,6 +239,13 @@ public void testCheckControlStructure() {
 		onBrickAtPosition(5)
 				.onCheckBox().check(matches(allOf(not(isChecked()), isEnabled())));
 	}
+	@Test
+	public void testAddToBackpackWithEmptyBrickSelection() {
+		openActionBarMenu();
+		onView(withText("Backpack")).perform(click());
```

Comment:
exchange withText("Backpack") with withText(R.string.backpack), so the test can work with different languages